### DISCUSSION TITLE
Implement snow for carla weather

### DISCUSSION THREAD

**Discussion by carlagx** [carla-simulator/carla#4852] [Ideas]
I create synthetic training datasets with carla and would be happy if snow would also work with PythonAPI. Is it planned to implement snow or am I rather an exception?

**Comment by EmiliyanGospodinov** [comment]
Hey, since I'm looking for the same, can you please share the dataset or at least tell me how you did that? 
You can save me a lot of time. 

I really appreciate any help you can provide.

**Reply by carlagx** [reply]
Hey, do you mean specifically the snow or in general creation of datasets? For snow I have a bluprint built into Carla 9.13, I can share that, but it is still a prototype, so not particularly performant. For datasets it is just a python + yaml file and also a prototyp

**Reply by EmiliyanGospodinov** [reply]
Hey, I'm specifically interested in the snow scenario, so maybe you can share the blueprint you've created, I'm just wondering how I can do the snow part and not so much in terms of efficiency or performance. 

Thank you very much in advance :)

**Reply by carlagx** [reply]
Here is https://github.com/carlagx/carla_snow the code, do not forget a lot of manual work necessary. Questions please via discord finn_gx#3249

**Reply by deyang2000** [reply]
It seems that the website doesn't work now.

**Reply by hank5000** [reply]
@carlagx  could you share your private repo for me ? , I am interested in the snow scenario too.

**Reply by carlagx** [reply]
send me your email

**Reply by hank5000** [reply]
> send me your email

@carlagx hankwuwu@gmail.com , thanks!

**Reply by SHITIANYU-hue** [reply]
Hi I am also interested, could you please also send me email tianyu.s@outlook.com ~

**Reply by Lizhe1228** [reply]
Hi I am also interested, could you please also send me email [lizhe99@hrbeu.edu.cn](lizhe99@hrbeu.edu.cn)

**Reply by VasilyBugaev** [reply]
Hi I am also interested, could you please also send me email [bugaevvasiliy@yandex.ru](bugaevvasiliy@yandex.ru)

**Reply by Anderw-S** [reply]
Hi I am also interested, could you please also send me email andrewsu317@gmail.com

**Reply by yuxinhi** [reply]
Hi I am also interested, could you please also send me email yuxingujing0420@gmail.com

really appreciate any help you can provide.

**Reply by SeniruD** [reply]
Hi I am also interested, could you please also send me email senirudissanayake@gmail.com

**Comment by carlagx** [comment]
Because of private repository, this is only a prototype

**Reply by AliSafaeifar** [reply]
Hi I am also interested, could you please also send me email a.safa.1372@gmail.com

**Reply by ashfakshibli** [reply]
Hello, I am interested in the snow scenario. If you please can add me shibli.emon@gmail.com

**Comment by StijnOosterlinck** [comment]
Are there any updates on this matter? 

Seems like snow is still not available...
